extensions/guacamole-auth-jdbc/modules/guacamole-auth-jdbc-base/pom.xml

```diff
@@ -116,7 +116,7 @@
         <dependency>
             <groupId>org.mybatis</groupId>
             <artifactId>mybatis-guice</artifactId>
-            <version>3.6</version>
+            <version>3.9</version>
```

Comment:
Needs discussion...

Reply:
I can probably do this without updating mybatis-guice, without the JdbcHelper support for SQLite, if this change needs to be a separate issue.  I was able to bump up both mybatis and mybatis-guice versions to the latest without seeing any adverse effects, but it probably warrants further discussion and testing.